test/line-visual-polar.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
What is this test case for?

Reply:
This is for other issues. I committed by mistake... Thanks for reminding!

Reply:
It seems package.json and package-lock.json are conflicting.